plugins/services/src/js/containers/tasks/TasksView.js

```diff
@@ -147,6 +147,10 @@ class TasksView extends mixin(SaveStateMixin) {
     );
   }
 
+  isSchedulerTask(task) {
+    return task.labels.some(({ key }) => key === "DCOS_PACKAGE_FRAMEWORK_NAME");
```

Comment:
So those are plan objects and not `Task` structs? How do you think is it better to put in into `TaskUtil`?

Reply:
Agree with @nLight although I would not block.